src/components/Toolbar/Toolbar.react.js

```diff
@@ -7,31 +7,59 @@
  */
 import PropTypes from 'lib/PropTypes';
 import React     from 'react';
+import Icon      from 'components/Icon/Icon.react';
 import styles    from 'components/Toolbar/Toolbar.scss';
+import history   from 'dashboard/history';
 
-let Toolbar = (props) => (
-  <div className={styles.toolbar}>
-    <div className={styles.title}>
-      <div className={styles.section}>{props.section}</div>
-      <div>
-        <span className={styles.subsection}>
-          {props.subsection}
-        </span>
-        <span className={styles.details}>
-          {props.details}
-        </span>
+const goBack = () => history.goBack();
```

Comment:
I know it matters. I just added this because relation viewer has url statement. If a user access relation viewer first, it will matter. But I have no idea to implement this better. Can you suggest about it?